tests/mappers/test_subworkflow_mapper.py

```diff
@@ -78,16 +78,11 @@ def test_create_mapper_jinja(self, parse_els):
         self.assertEqual("test_id", mapper.task_id)
         self.assertEqual(TriggerRule.DUMMY, mapper.trigger_rule)
         self.assertEqual(self.subworkflow_node, mapper.oozie_node)
-        self.assertEqual(
-            {
-                "examplesRoot": "examples",
-                "nameNode": "hdfs://",
-                "oozie.wf.application.path": "hdfs:///user/pig/examples/pig",
-                "resourceManager": "localhost:8032",
-                "user.name": "potiuk",
-            },
-            mapper.property_set.job_properties,
-        )
+        self.assertEqual("examples", mapper.property_set["examplesRoot"])
+        self.assertEqual("hdfs://", mapper.property_set["nameNode"])
+        self.assertEqual("hdfs:///user/pig/examples/pig", mapper.property_set["oozie.wf.application.path"])
+        self.assertEqual("localhost:8032", mapper.property_set["resourceManager"])
+        self.assertIsNotNone(mapper.property_set["user.name"])
```

Comment:
Do you think it's worth adding a test to check the keys in the dictionary?  Now a new key may appear, which will not be checked in any test.

Reply:
I think so. We are checking specific mapper as an example, so we can expect that those properties are in the property_set.